src/register.js

```diff
@@ -236,8 +239,13 @@ window.happo.nextExample = async () => {
     waitFor,
     beforeScreenshot,
     theme,
+    axeOptions,
   } = examples[currentIndex];
 
+  // Set the axe configuration for the current story globally so that the
+  // Happo worker can pick it up
+  window.happoAxeOptions = axeOptions;
```

Comment:
Global seems okay.

But I wonder if we should think about attaching a version to this somehow so that if we want to ship a big axe update that changes theway that options works we have a way to avoid breaking everyone.

Reply:
Perhaps -- How would that look?